cmd/mattermost/commands/command.go

```diff
@@ -67,3 +76,33 @@ func moveCommandCmdF(command *cobra.Command, args []string) error {
 func moveCommand(a *app.App, team *model.Team, command *model.Command) *model.AppError {
 	return a.MoveCommand(team, command)
 }
+
+func listCommandCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+
+	if len(args) < 1 {
+		return errors.New("Enter at least one team")
+	}
+
+	teams := getTeamsFromTeamArgs(a, args)
+	for i, team := range teams {
+		if team == nil {
+			CommandPrintErrorln("Unable to find team '" + args[i] + "'")
+			continue
+		}
+		if result := <-a.Srv.Store.Command().GetByTeam(team.Id); result.Err != nil {
```

Comment:
We like to use idiomatic error handling as much as possible:

For example here, we would use something like:
```go
result := <-a.Srv.Store.Command().GetByTeam(team.Id)
if result.Err != nil {
    CommandPrintErrorln("Unable to list commands for '" + args[i] + "'")
    continue
}

commands := result.Data.([]*model.Command)
for _, command := range commands {
        CommandPrettyPrintln(command.DisplayName)
}
```